test/src/queryhandler.spec.ts

```diff
@@ -94,6 +94,85 @@ describe('Query handler tests', function () {
     });
   });
 
+  describe('Text selectors', function () {
+    describe('in Page', function () {
+      it('should query existing element', async () => {
```

Comment:
let's add the following tests:

1) deeply nested text `<div><div><span>text` to make sure that <span> is returned.
2) tests involving 'checkbox', 'image', 'radio' and an input (e.g., textarea or text)
3) test for the searched text spanning multiple element (`<div><span>a</span><span>b</span><div>` && `text/ab`)
4) let's test the order in which elements are found if there are multiple matches